### PR TITLE
Added Matrix Operations file

### PR DESCRIPTION
**Pull request by Wittty-Panda**
Added matrix operations using 2-D array program in c++ language in array folder
The program contains operations like addition, subtraction, multiplication, and transpose of matrix all using 2-D array concept.

### CODE REVIEW

File: Arrays/matrixoperationusing2d.cpp [hunk 1]
```diff
@@ -0,0 +1,231 @@
+//Operations on matrix using 2-D array
+//standred run should look like this
```
Comment (SidJain1412) [inline Arrays/matrixoperationusing2d.cpp:2]:
```suggestion
//standard run should look like this:
```

File: Arrays/matrixoperationusing2d.cpp [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+3  4  
+
+Matrix of Sum of two matrix is: 
+
+
+ Matrix is: 
+2  4  
+6  8  
+
+Matrix of Substraction of two matrix is:
```
Comment (SidJain1412) [inline Arrays/matrixoperationusing2d.cpp:38]:
```suggestion
Matrix of Subtraction of two matrix is: 
```

File: Arrays/matrixoperationusing2d.cpp [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+      int m[0][0];
+    };
+    Matrix(int,int);
+    Matrix(Matrix &x);
+    void displaydata(Matrix);
+    void add(Matrix,Matrix);
+    void sub(Matrix,Matrix);
+    void mul(Matrix, Matrix);
+    void trns(Matrix);
+    ~Matrix(){
```
Comment (SidJain1412) [inline Arrays/matrixoperationusing2d.cpp:80]: Can you please explain this class further?

File: Arrays/matrixoperationusing2d.cpp [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+    Matrix m7;
+    m7.trns(m1);
+    cout << endl << "Transpose of matrix is: " << endl;
+    m7.displaydata(m6);
+}
+
+
+
+
+
```
Comment (SidJain1412) [inline Arrays/matrixoperationusing2d.cpp:231]:
```suggestion
```

**Review by SidJain1412 — CHANGES_REQUESTED** [review]
Please have a look at the suggestions and fix/ clarify

File: Arrays/matrixoperationusing2d.cpp (hunk 3 above)
Comment (Wittty-Panda) [reply Arrays/matrixoperationusing2d.cpp:80]: it is just used as a destructor

**Comment by SidJain1412** [comment]
@Wittty-Panda thanks for contributing!

**Comment by Wittty-Panda** [comment]
@SidJain1412 happy to help
please tell if any contribution is needed

**Comment by Wittty-Panda** [comment]
also this is showing Ineligible Repository kind of mood off but anyways

**Comment by SidJain1412** [comment]
@all-contributors please add @Wittty-Panda for code

**Comment by Wittty-Panda** [comment]
thank you

**Comment by SidJain1412** [comment]
@Wittty-Panda not sure why it's ineligible. Can you help me figure that out? 

My bad otherwise I'll remove the hacktober tag so as to not mislead

**Comment by Wittty-Panda** [comment]
you had to add hacktoberfest-accepted to pr to make it valid

**Comment by Wittty-Panda** [comment]
don't worry I will add another program of use of template function
do you need that? it will be a concept in generic programming

**Comment by Wittty-Panda** [comment]
I talked with the folks at discord they said changing label might help @SidJain1412

**Comment by SidJain1412** [comment]
@Wittty-Panda my bad, added the label

**Pull request merged**